src/GraphQLParser/Visitors/IWriteContext.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using System.IO;
+
+namespace GraphQLParser.Visitors
+{
+    public interface IWriteContext : IVisitorContext
+    {
+        TextWriter Writer { get; }
+
+        Stack<AST.ASTNode> Parent { get; }
```

Comment:
```suggestion
        Stack<AST.ASTNode> Parents { get; }
```

Reply:
At first I called the property in this way.

Reply:
done